messages.js

```diff
@@ -0,0 +1,12 @@
+const pasteTextFi =
+  'Valitse Netbeansin ylävalikosta "TMC" -> "Send code to TMC Pastebin" ja valitse avautuvasta ikkunasta "Send". Tämän jälkeen saat linkin koodiisi, jonka voit kopioida ja liittää tänne.'
+const ideLogTextFi =
+  'Saat Netbeanssin IDE login näin: 1. Avaa Netbeans. 2. Aiheuta ongelma uudestaan. 3. Valitse Netbeanssin yläpalkista "View" -> "IDE Log". 4. Kopioi avautuvasta paneelista kaikki teksti ja liitä se vastaukseesi.'
+const chmodTextFi =
+  'Suorita päätteessä komento chmod +x /Users/ruusukivi/.config/tmc/apache-maven-3.5.4/bin/mvn Pääte/komentorivi löytyy painamalla command + välilyönti ja kirjoittamalla Pääte (eng. Terminal)'
```

Comment:
```suggestion
  'Suorita päätteessä komento `chmod +x $HOME/.config/tmc/apache-maven-3.5.4/bin/mvn` Pääte/komentorivi löytyy painamalla command + välilyönti ja kirjoittamalla Pääte (eng. Terminal)'
```

Toimiikohan toi ` code/block format bottien kanssa. Mut ainakin toi $HOME niin ei tarvitse arpoa käyttäjätunnusta.

Onko maven versio aina sama, vai pitäisikö tosta vielä vaihtaa komennoksi:
```
chmod +x $HOME/.config/tmc/apache-maven-*/bin/mvn`
```

Reply:
oho, tais jäädä toi suora vastaus telegramista. Home olisi kyllä parempi

Reply:
> Onko maven versio aina sama, vai pitäisikö tosta vielä vaihtaa komennoksi:

Maven-versio ei pitäisi muuttua koska tuohon pitäisi tulla automaattinen korjaus ennen seuraavaa mavenin päivitystä.

Itselleni tuli tosta mieleen että olen törmännyt typpeihin joille on epäselvää että komento ajetaan terminaalissa enterillä. Ehkä loppuun voisi vielä jatkaa: "Saat komennon ajettua liittämällä/kirjoittamalla sen avautuvaan ikkunaan ja painamalla enter."